genCode.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, division, print_function, \
+    unicode_literals
+
+from io import open
+
+import json
+import os
+import re
+import six
+import textwrap
+
+from jinja2 import Environment, FileSystemLoader
+
+GENERATED_STRING = "# This file is generated!  Do not edit!"
+# ROUTING_KEY_WHITELIST is also defined in test/base/__init__.py
+# if you change it, please update it there as well.
+ROUTING_KEY_WHITELIST = ("name", "multipleWords", "constant")
```

Comment:
If the purpose of this is to remove a couple specific keys, I think a blacklist would be better.